sdk/formrecognizer/Azure.AI.FormRecognizer/MigrationGuide.md

```diff
@@ -0,0 +1,739 @@
+
+                            Console.WriteLine($"  UnitPrice: '{itemUnitPrice}', with confidence {itemUnitPriceField.Confidence}");
+                        }
+                    }
+
+                    if (itemFields.TryGetValue("Quantity", out FormField itemQuantityField))
+                    {
+                        if (itemQuantityField.Value.ValueType == FieldValueType.Float)
+                        {
+                            float quantityAmount = itemQuantityField.Value.AsFloat();
+
+                            Console.WriteLine($"  Quantity: '{quantityAmount}', with confidence {itemQuantityField.Confidence}");
+                        }
+                    }
+
+                    if (itemFields.TryGetValue("Amount", out FormField itemAmountField))
+                    {
+                        if (itemAmountField.Value.ValueType == FieldValueType.Float)
+                        {
+                            float itemAmount = itemAmountField.Value.AsFloat();
+
+                            Console.WriteLine($"  Amount: '{itemAmount}', with confidence {itemAmountField.Confidence}");
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    if (invoice.Fields.TryGetValue("SubTotal", out FormField subTotalField))
+    {
+        if (subTotalField.Value.ValueType == FieldValueType.Float)
+        {
+            float subTotal = subTotalField.Value.AsFloat();
+            Console.WriteLine($"Sub Total: '{subTotal}', with confidence {subTotalField.Confidence}");
+        }
+    }
+
+    if (invoice.Fields.TryGetValue("TotalTax", out FormField totalTaxField))
+    {
+        if (totalTaxField.Value.ValueType == FieldValueType.Float)
+        {
+            float totalTax = totalTaxField.Value.AsFloat();
+            Console.WriteLine($"Total Tax: '{totalTax}', with confidence {totalTaxField.Confidence}");
+        }
+    }
+
+    if (invoice.Fields.TryGetValue("InvoiceTotal", out FormField invoiceTotalField))
+    {
+        if (invoiceTotalField.Value.ValueType == FieldValueType.Float)
+        {
+            float invoiceTotal = invoiceTotalField.Value.AsFloat();
+            Console.WriteLine($"Invoice Total: '{invoiceTotal}', with confidence {invoiceTotalField.Confidence}");
+        }
+    }
+}
+```
+
+Analyzing prebuilt models like business cards, identity documents, invoices, and receipts with `4.0.x`:
+```C# Snippet:FormRecognizerAnalyzeWithPrebuiltModelFromUriAsync
+string fileUri = "<fileUri>";
+
+var options = new AnalyzeDocumentOptions() { Locale = "en-US" };
+
+AnalyzeDocumentOperation operation = await client.StartAnalyzeDocumentFromUriAsync("prebuilt-invoice", fileUri, options);
+
+await operation.WaitForCompletionAsync();
+
+AnalyzeResult result = operation.Value;
+
+// To see the list of all the supported fields returned by service and its corresponding types for the
+// prebuilt-invoice model, consult:
+// https://aka.ms/formrecognizer/invoicefields
```

Comment:
invoicefieldschema

Reply:
This was updated in master, so I had to update my branch.
Thanks for the reminder!